forum/users/serializers.py

```diff
@@ -70,3 +71,31 @@ def validate_role(self, value):
         if value not in valid_roles:
             raise serializers.ValidationError("Invalid role.")
         return value
+
+
+VALID_TOKEN_ROLES = [Role.STARTUP, Role.INVESTOR]
+
+class CustomTokenObtainPairSerializer(TokenObtainPairSerializer):
```

Comment:
The `role` field is required for authentication, but currently, only `email` and `password` are visible in Swagger by default. Please add the `role` field to the Swagger schema for the login endpoint.  

For example, you can add
```python
role = serializers.IntegerField(required=True)
```  
before 'validate' method at line 79.

Reply:
fr, done ✅